runtime/onert/core/src/ir/Shape.cc

```diff
@@ -30,7 +30,7 @@ int32_t const Shape::UNSPECIFIED_DIM = -1;
 
 FeatureShape Shape::asFeature(Layout layout) const
 {
-  assert(rank() == 4);
+  //assert(rank() == 4);
```

Comment:
In case of dynamic tensor, when using asFeature in kernelGenerator, rank is not known and an error occurs. How do I solve this?